master/README.md

```diff
@@ -75,7 +75,7 @@ See [the list of releases][6] to find out about feature changes.
 
 
 [29]: https://heptio.github.io/velero/
-[30]: /troubleshooting.md
+[30]: /docs/troubleshooting.md
```

Comment:
I think these links need to be tweaked manually to not have `/docs` in them, right?